src/core_init_atmosphere/mpas_geotile_manager.F

```diff
@@ -0,0 +1,1038 @@
+        type (mpas_geotile_type), pointer, intent(in) :: tile
+        integer, value :: j
+        integer, value :: i
+        real (kind=RKIND), intent(out) :: lat
+        real (kind=RKIND), intent(out) :: lon
+
+        integer, pointer :: tile_bdr
+        real (kind=RKIND), pointer :: known_lon
+        real (kind=RKIND), pointer :: known_lat
+        real (kind=RKIND), pointer :: dx
+        real (kind=RKIND), pointer :: dy
+        integer :: ierr
+
+        ierr = 0
+
+        call mpas_pool_get_config(mgr % pool, 'tile_bdr', tile_bdr)
+        call mpas_pool_get_config(mgr % pool, 'known_lat', known_lat)
+        call mpas_pool_get_config(mgr % pool, 'known_lon', known_lon)
+        call mpas_pool_get_config(mgr % pool, 'dx', dx)
+        call mpas_pool_get_config(mgr % pool, 'dy', dy)
+
+        lat = real((j - (tile_bdr + 1) + tile % y - 1), kind=RKIND) * dy + known_lat
+        lon = real((i - (tile_bdr + 1) + tile % x - 1), kind=RKIND) * dx + known_lon
+
+        call deg2Rad(lat)
+        call deg2Rad(lon)
+
+    end subroutine mpas_geotile_mgr_tile_to_latlon
+
+
+    !***********************************************************************
+    !
+    !  public subroutine mpas_geotile_mgr_latlon_to_pixel => latlon_to_pixel
+    !
+    !> \brief   Translate a latitude, longitude coordinate to pixel coordinates
+    !> \author  Miles A. Curry
+    !> \date    02/20/2020
+    !> \details
+    !> Translate a latitude, longitude coordinate into global pixel coordinates. lat
+    !> should be in the range of -.5 * pi to .5 * pi and lon should be between -pi
+    !> and pi.
+    !
+    !-----------------------------------------------------------------------
+    subroutine mpas_geotile_mgr_latlon_to_pixel(mgr, lat, lon, pixel_x, pixel_y)
+
+        implicit none
+
+        class (mpas_geotile_mgr_type) :: mgr
+        real (kind=RKIND), value :: lat
+        real (kind=RKIND), value :: lon
+        integer, intent(out) :: pixel_x
+        integer, intent(out) :: pixel_y
+
+        integer, pointer :: tile_bdr
+        real (kind=RKIND), pointer :: known_lon
+        real (kind=RKIND), pointer :: known_lat
+        real, pointer :: dx
+        real, pointer :: dy
+        integer :: ierr
+
+        ierr = 0
+
+        call mpas_pool_get_config(mgr % pool, 'tile_bdr', tile_bdr)
+        call mpas_pool_get_config(mgr % pool, 'known_lon', known_lon)
+        call mpas_pool_get_config(mgr % pool, 'known_lat', known_lat)
+        call mpas_pool_get_config(mgr % pool, 'dx', dx)
+        call mpas_pool_get_config(mgr % pool, 'dy', dy)
+
+        call rad2Deg(lat)
+        call rad2Deg(lon)
+
+        pixel_x = nint((lon - known_lon) / dx)
+        pixel_y = nint((lat - known_lat) / dy)
+
+        if (pixel_x < 0) then
+            pixel_x = pixel_x + mgr % pixel_nx
+        endif
+
+        if (pixel_y < 0) then
+            pixel_y = 0
+        else if (pixel_y >= mgr % pixel_ny) then
+            pixel_y = pixel_y - 1
```

Comment:
Should `pixel_y` be `mgr % pixel_ny` on the r.h.s.?

Reply:
I think you're right. I'll push an update.